### PR TITLE
Dark UI mode

### LINKED ISSUE #50: Switch to dark mode
Restyle the UI into a dark color scheme.

### PR DESCRIPTION
**Pull request by bzurkowski**
Applies the dark color scheme to all UI components.

Closes #50.

### CODE REVIEW

**Comment by bzurkowski** [comment]
@k-jano @filwie How do you like the dark mode? 😉 

Screenshot 2020-07-12 at 14 27 57

Can you help with coloring the date picker and namespace selector?

File: src/components/Graph.scss [hunk 1]
```diff
@@ -5,6 +5,7 @@ svg.graph {
     top     : 0;
     left    : 0;
     z-index : 5;
+    background: #15181a
```
Comment (filwie) [inline src/components/Graph.scss:8]: can we create a variable for this color in _variables.scss?

File: src/components/NodeDetailCard.scss [hunk 2]
```diff
@@ -1,36 +1,41 @@
...
-.node-info-card {
+
+.card.node-info-card {
     width     : 23rem;
     z-index   : 1030;
     position  : relative;
     opacity   : 0.9;
     transition: visibility opacity 500ms linear;
     margin    : 2rem 0 0 2rem;
+    background-color: #252a2d;
```
Comment (filwie) [inline src/components/NodeDetailCard.scss:11]: A variable in `_variables.scss` would be nice :)

File: src/components/NodeDetailCard.scss [hunk 3]
```diff
@@ -1,36 +1,41 @@
 @import "_variables.scss";
 
-.node-info-card {
+
+.card.node-info-card {
```
Comment (filwie) [inline src/components/NodeDetailCard.scss:4]: Let's keep one convention - either with a space between classes or without it

**Comment by k-jano** [comment]
I really enjoy presented dark mode :smiley: ! Please refer @filwie comments

**Pull request merged**